packages/flutter/test_fixes/material.dart

```diff
@@ -518,4 +518,48 @@ void main() {
   colorScheme = colorScheme.copyWith(primaryVariant: Colors.black, secondaryVariant: Colors.white);
   colorScheme.primaryVariant;
   colorScheme.secondaryVariant;
+
+  // Changes mades in https://github.com/flutter/flutter/pull/94156
+  ThemeData themeData = ThemeData();
+  themeData = ThemeData(errorColor: Colors.red);
+  themeData = ThemeData(errorColor: Colors.red, primarySwatch: Colors.blue);
+  themeData = ThemeData(errorColor: Colors.red, colorScheme: ColorScheme.light());
+  themeData = ThemeData(errorColor: Colors.red, colorScheme: ColorScheme.light(), primarySwatch: Colors.blue);
+  themeData = ThemeData(error: '');
+  themeData = ThemeData.raw(errorColor: Colors.red);
+  themeData = ThemeData.raw(errorColor: Colors.red, primarySwatch: Colors.blue);
+  themeData = ThemeData.raw(errorColor: Colors.red, colorScheme: ColorScheme.light());
+  themeData = ThemeData.raw(errorColor: Colors.red, colorScheme: ColorScheme.light(), primarySwatch: Colors.blue);
+  themeData = ThemeData.raw(error: '');
+  themeData = themeData.copyWith(errorColor: Colors.red);
+  themeData = themeData.copyWith(error: '');
+  themeData = themeData.copyWith(errorColor: Colors.red, primarySwatch: Colors.blue);
+  themeData = themeData.copyWith(errorColor: Colors.red, colorScheme: ColorScheme.light());
+  themeData = themeData.copyWith(errorColor: Colors.red, colorScheme: ColorScheme.light(), primarySwatch: Colors.blue);
+  themeData.errorColor;
+
+  // Changes mades in https://github.com/flutter/flutter/pull/94156
+  ThemeData themeData = ThemeData();
+  themeData = ThemeData(backgroundColor: Colors.grey);
+  themeData = ThemeData(backgroundColor: Colors.grey, primarySwatch: Colors.blue);
+  themeData = ThemeData(backgroundColor: Colors.grey, colorScheme: ColorScheme.light());
+  themeData = ThemeData(backgroundColor: Colors.grey, colorScheme: ColorScheme.light(), primarySwatch: Colors.blue);
+  themeData = ThemeData(error: '');
+  themeData = ThemeData.raw(backgroundColor: Colors.grey);
+  themeData = ThemeData.raw(backgroundColor: Colors.grey, primarySwatch: Colors.blue);
+  themeData = ThemeData.raw(backgroundColor: Colors.grey, colorScheme: ColorScheme.light());
+  themeData = ThemeData.raw(backgroundColor: Colors.grey, colorScheme: ColorScheme.light(), primarySwatch: Colors.blue);
+  themeData = ThemeData.raw(error: '');
+  themeData = themeData.copyWith(backgroundColor: Colors.grey);
+  themeData = themeData.copyWith(error: '');
+  themeData = themeData.copyWith(backgroundColor: Colors.grey, primarySwatch: Colors.blue);
+  themeData = themeData.copyWith(backgroundColor: Colors.grey, colorScheme: ColorScheme.light());
+  themeData = themeData.copyWith(backgroundColor: Colors.grey, colorScheme: ColorScheme.light(), primarySwatch: Colors.blue);
+  themeData.backgroundColor;
+
+  // Changes mades in https://github.com/flutter/flutter/pull/94156 TODO: REMOVE
```

Comment:
@Piinks Hey, I'd love your help here. I adapted the exising migrations from `ThemeData.accentColor` to `ColorScheme.secondary` for this. The first two sets of migrations work fine. However, I was curious about the case where both `backgroundColor` and `errorColor` are currently defined. Rather than move both to a `colorScheme`, the result is:

```
...
 ThemeData themeData = ThemeData();
  themeData = ThemeData(backgroundColor: Colors.grey, colorScheme: ColorScheme.fromSwatch().copyWith(error: Colors.red));
  themeData = ThemeData.raw(backgroundColor: Colors.grey, colorScheme: ColorScheme.fromSwatch().copyWith(error: Colors.red));
  themeData = themeData.copyWith(backgroundColor: Colors.grey, colorScheme: ColorScheme.fromSwatch().copyWith(error: Colors.red));
}

Edits returned from server:
  /Users/plg/development/flutter-dev/packages/flutter/test_fixes/material.dart
    25736 - 25736, ', colorScheme: ColorScheme.fromSwatch().copyWith(error: Colors.red)'
    25712 - 25736, ''
    25648 - 25648, ', colorScheme: ColorScheme.fromSwatch().copyWith(error: Colors.red)'
    25624 - 25648, ''
    25565 - 25565, ', colorScheme: ColorScheme.fromSwatch().copyWith(error: Colors.red)'
...
```

I wondering about how to change the migrations to make them powerful enough to handle multiple deprecations (as will be the case in future releases)

Reply:
Hey @guidezpl apologies for keeping you waiting, I was OOO. I'm looking into this now. Thanks for writing fixes!